app/core/ai_cache.py

```
@@ -0,0 +1,106 @@
+                key TEXT PRIMARY KEY,
+                access_count INTEGER,
+                last_access TIMESTAMP,
+                weight REAL
+            )
+        """)
+
+    def get(self, key: str) -> Optional[Any]:
+        with self._lock:
+            if key in self._cache:
+                item = self._cache[key]
+                if item.ttl and datetime.now() - item.created_at > timedelta(seconds=item.ttl):
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)